client/src/component/TopBar.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+
+import DragHandleIcon from '@material-ui/icons/DragHandle';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import clsx from 'clsx';
+import { Link } from "react-router-dom";
+
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		flexGrow: 1,
+	},
+	menuButton: {
+		marginRight: theme.spacing(2),
+	},
+	title: {
+		flexGrow: 1,
+	},
+	AppBar: {
+		backgroundColor: '#fff',
+		color: '#000',
+	},
+}));
+
+
+
+export default function TopBar(props) {
+
+	const classes = useStyles();
+	const user = props.user
+	const [auth, setAuth] = React.useState(user);
```

Comment:
see note above. No need to keep doing `React.useState()`.

Reply:
Also it is an antipattern to duplicate a prop with local state. You can just access the prop, there's no need for the `auth` state here.